patroni/postgresql/bootstrap.py

```diff
@@ -98,7 +102,8 @@ def _post_restore(self):
 
     def _custom_bootstrap(self, config):
         self._postgresql.set_state('running custom bootstrap script')
-        params = ['--scope=' + self._postgresql.scope, '--datadir=' + self._postgresql.data_dir]
+        params = [] if not self.supply_params else ['--scope=' + self._postgresql.scope,
+                                                    '--datadir=' + self._postgresql.data_dir]
```

Comment:
The `config` argument has everything we need:
```suggestion
        params = [] if config.get('no_params') else\
                ['--scope=' + self._postgresql.scope, '--datadir=' + self._postgresql.data_dir]
```

Any other change could go.

Reply:
Thank you. Code updated